using-valtimo/plugin/documenten-api/configure-plugin.md

```diff
@@ -0,0 +1,53 @@
+# Using the Documenten API Plugin
+
+The Documenten API plugin can be used to store documents and their metadata in an application that supports the
```

Comment:
Instead of **can be**, use **is**. This is what it is meant for. Also missing an example here, but not sure if this is really an issue.